Cli-Shared/Program.cs

```diff
@@ -1113,6 +1215,35 @@ private static bool QueryCredentials(OperationArguments operationArguments)
                     }
                     break;
 
+                case AuthorityType.Bitbucket:
+					{
+                    var bbcAuth = authentication as Bitbucket.Authentication;
+
+                    Task.Run(async () =>
+                    {
+                        if (((operationArguments.Interactivity != Interactivity.Always)
+                             && ((credentials = bbcAuth.GetCredentials(operationArguments.TargetUri, operationArguments.CredUsername)) != null)
+                             && (!operationArguments.ValidateCredentials
+                                 || ((credentials = await bbcAuth.ValidateCredentials(operationArguments.TargetUri, operationArguments.CredUsername, credentials)) != null)))
+                            || ((operationArguments.Interactivity != Interactivity.Never)
+                                && ((credentials = await bbcAuth.InteractiveLogon(operationArguments.TargetUri, operationArguments.CredUsername)) != null)
+                                && (!operationArguments.ValidateCredentials
+                                    || ((credentials = await bbcAuth.ValidateCredentials(operationArguments.TargetUri, operationArguments.CredUsername, credentials)) != null))))
+                        {
+                                Git.Trace.WriteLine($"credentials for '{operationArguments.TargetUri}' found.");
+                                operationArguments.SetCredentials(credentials);
```

Comment:
The "Personal Access Token" thing is just a placeholder for GitHub and VSTS token values. Bitbucket can issue whatever it wants there. Neither GitHub nor VSTS will place the actual username into the saved token credential however. :simple_smile: